contents/blog/duolingo-testing-lessons.md

```diff
@@ -0,0 +1,107 @@
+---
+title: What can we learn from how Duolingo uses A/B tests?
+date: 2024-09-24
+author:
+  - bijan-boustani
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/blog/experiment-hog.jpg
+featuredImageType: full
+tags:
+  - Experiments
+  - Product engineers
+crosspost:
+  - Blog
+---
+
+Duolingo has grown rapidly over the past few years, with a market cap recently exceeding $10B. ["Test everything"](https://blog.duolingo.com/improving-duolingo-one-experiment-at-a-time/) is one of their core principles, and they credit **A/B testing** as _the_ source of the company's success. [source](https://www.youtube.com/watch?v=9KqrnBiyBQ8)
+
+In this article, we'll try to extract a few lessons from Duolingo's experimentation process.
+
+- reframe the way we think about our product, focusing on tiny details to continually improve quality
+- build a massive audience by offering a generous free tier
+- improve onboarding, retention, and user engagement
+- align the goals of our users with the goals of our company
```

Comment:
I still think this intro is lacking a little... edge? This is a hard thing to articulate, but have a read through some recent newsletters we've done for inspo. Some good examples:

- https://newsletter.posthog.com/p/how-not-to-be-boring
- https://newsletter.posthog.com/p/how-to-get-a-job-at-a-startup
- https://newsletter.posthog.com/p/hiring-and-managing-cracked-engineers

Don't feel like you need to copy these, but I hope they convey the vibe we want to find.

Reply:
Totally agree. I see what you mean about those other posts having an edge and authenticity. I tried adding a handful of changes here, but I'm open to some more drastic updates as well:
https://github.com/PostHog/posthog.com/pull/9423/commits/373ac0e1de8a743b61cfc934fc2966901fd14f90

Reply:
And the article in Italian.

Reply:
And the CTA in Chinese.